test/system/static.test.ts

```diff
@@ -308,10 +308,10 @@ test("able to scan opensuse/leap images", async (t) => {
  *
  * Even though the node "binary"
  * (actually just an empty file to keep the fixture small)
- * has been removed, we still report it exists.
+ * has been removed, we used to report it exists.
  */
 
-test("bug exists: false positive for deleted files", async (t) => {
+test("bug fixed: used not to recognize deleted files", async (t) => {
```

Comment:
I don't think we need that test any more, can remove, we have the functionality covered now.